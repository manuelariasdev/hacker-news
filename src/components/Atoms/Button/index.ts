export * from './Button'
export * from './buttonStyle'