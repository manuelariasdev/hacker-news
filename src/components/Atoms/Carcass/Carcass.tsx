import * as React from 'react'

import { StyledCarcass } from './carcassStyle'

export type CarcassProps = React.CSSProperties;

export const Carcass = ({ height }: CarcassProps) => {
  return <StyledCarcass height={height} />
}


