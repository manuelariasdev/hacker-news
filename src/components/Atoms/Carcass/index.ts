export * from './Carcass'
export * from './carcassStyle'