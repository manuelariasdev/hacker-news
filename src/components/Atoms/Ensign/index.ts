export * from './Ensign'
export * from './ensignStyles'