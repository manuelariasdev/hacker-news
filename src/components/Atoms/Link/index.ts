export * from './Link'
export * from './linkStyle'