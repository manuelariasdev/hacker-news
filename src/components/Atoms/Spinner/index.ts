export * from './Spiner'
export * from './spinerStyles'