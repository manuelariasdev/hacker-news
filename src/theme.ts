
import { DefaultTheme } from 'styled-components'

const theme: DefaultTheme = {
  borderRadius: {
    sm: '3px',
    md: '6px',
    lg: '15px',
    xl: '50px'
  },
  boxShadow: {
    sm: ' 0 1px 4px 0 rgba(0, 21, 41, 0.12)'
  },
  fontSize: {
    xs: '0.68rem', 
    sm: '0.875rem', 
    md: '16px',
    lg: '1.25rem', 
    xl: '1.75rem'
  },
  letterSpacing: {
    md: 0.25
  },
  lineHeight: {
    xs: '1rem',
    sm: '1.25rem',
    md: '16px',
    lg: '1.25rem',
    xl: '1.75rem'
  },
  transition: 'all 700ms cubic-bezier(0.15, 1, 0.20, 1)'
}

export default theme
