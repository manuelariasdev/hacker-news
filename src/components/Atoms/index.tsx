import { Ensign } from './Ensign'
import { Button } from './Button'
import { Icon } from './Icon'
import { IconButton } from './IconButton'
import { Link } from './Link'
import { Carcass } from './Carcass'
import { Spinner } from './Spinner'
import { Text } from './Text'
import { Title } from './Title'


export {
  Ensign,
  Button,
  Icon,
  IconButton,
  Link,
  Carcass,
  Spinner,
  Text,
  Title
}
