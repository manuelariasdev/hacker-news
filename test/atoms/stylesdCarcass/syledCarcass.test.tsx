/**
 * @jest-environment jsdom
 */
 import * as React from "react";
 import theme from '../../../theme';
 import { render, screen } from '@testing-library/react'
 import {StyledCarcass } from "../../../src/components/Atoms/Carcass"
 
 describe('Title <Title/>', function () {
 
       test('debe de hacer match con el snapshot', () => {
           
         const {container} = render(<StyledCarcass theme={theme}/>)
         expect( container ).toMatchSnapshot();
         })
    
 });