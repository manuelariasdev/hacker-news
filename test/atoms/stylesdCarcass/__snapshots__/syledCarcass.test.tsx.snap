// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title <Title/> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="sc-bczRLJ ijhGnZ"
  />
</div>
`;
